### ISSUE TITLE
Keyboard navigation fails when iframe content is focused

### ISSUE THREAD

**Issue by chocolatkey** [readium/ts-toolkit#221]
In the playground, for both reflowable (mobyd dick) and fxl (bella) publications, the keyboard navigation seems bugged. When you first load a publication, it works fine, but if you click in the content (focusing it), then you can only navigate one more page before the arrow keys just stop working. Every time you click (focus) the content again, you gain one more arrow key press before it breaks

https://github.com/user-attachments/assets/80828939-5781-4634-a8f1-c94ea3952877

**Comment by JayPanoz** [comment]
That was less trivial than anticipated because the way Thorium Web previously handled peripherals kinda brute-forced its way around the dual context of navigator (parent document, iframe). 

Long story short: 

- the event was swallowed by unfocused iframes, because they actually kept the `activeElement`, so the navigator never got these messages
- I'm also removing listeners on unfocus to make sure they do not intercept key combinations they should not as another precaution
- when navigating too rapidly, listeners could not even be plugged properly and it would break entirely. I added a navigation lock to prevent rapid-fire navigation, which also treats #190 

We'll really need to do additional work on focus management tho. One thing we know is there is a need to handle tab + snapping, but also escaping iframes, etc.
